test/JDBC/expected/case_insensitive_collation-before-13-5-vu-prepare.out

```diff
@@ -0,0 +1,38 @@
+-- BABEL-3094
+-- sys table and view columns need to have the correct output type and collation
+create table t_rcv15 ([Test] varchar);
+go
+
+CREATE TABLE CaMeLtAbLe(cAmElCoLuMn VARCHAR(100) NOT NULL);
+go
+
+CREATE VIEW CaMeLvIeW AS SELECT 'VIEW' + cAmElCoLuMn + 'VIEW' AS CaMeLcOlUmNiNvIeW FROM CaMeLtAbLe;
+go
+
+-- BABEL-3165
+-- sys.databases.name should have case-insensitive collation
+create database TEST;
+go
+
+-- BABEL-3323
+-- When any sys view contains collatable constants
+create view vB as ( SELECT CASE WHEN 'b' = 'B' THEN 'case-insensitive' ELSE 'case-sensitive' END);
+go
+
+-- [BABEL-526]
+-- When creating an UDD and then using it for a column datatype,
+-- before fix, it used to show error 'type "dbo.mytyp6" does not exist
+create type [dbo].[myTyp6] from int;
+go
+
+create table t_rcv16 (a [dbo].[myTyp6]);
+go
+
+create table t_rcv17(a dbo.myTyp6);
+go
```

Comment:
These should be treated as verification tests. Please move these test in verify file because we want to show that we should be able to use UDD case insensitive way.

Reply:
Original discussion - https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/930/files#r1048003553

Reply:
shifted to verify files